ignore.go

```diff
@@ -0,0 +1,44 @@
+package git
+
+/*
+#include <git2.h>
+*/
+import "C"
+import (
+	"runtime"
+)
+
+func (v *Repository) AddInternalIgnoreRules(rules string) error {
```

Comment:
This name doesn't match the libgit2 name. It should also not be plural, as it only accepts a single rule to add.